termwiz/src/tmux_cc/mod.rs

```diff
@@ -961,7 +966,7 @@ mod tests {
                 number: 321,
                 flags: 0,
             },
-            parse_line("%begin 12345 321 0").unwrap()
+            parse_line("%begin 12345 321 0".to_owned().as_bytes()).unwrap()
```

Comment:
I think you can write this more simply like this, and use the same technique for the other cases below:

```suggestion
            parse_line(b"%begin 12345 321 0").unwrap()
```